system/shaders/output_d3d.fx

```diff
@@ -194,7 +194,7 @@ float4 OUTPUT_PS(VS_OUTPUT In) : SV_TARGET
 {
   float4 color = g_Texture.Sample(KernelSampler, In.TextureUV);
   [flatten] if (m_params.x)
-    color = saturate(0.0625 + color * 219.0 / 255.0);
+    color = saturate((64.0 / 1023.0) + color * (940.0 - 64.0) / 1023.0);
```

Comment:
Is it OK for this one not to take the number of bits of the output into account?